dds_web/utils.py

```diff
@@ -95,7 +104,7 @@ def invoice_units():
     # From safespring
     old_file = parent_dir / pathlib.Path("development/invoicing/safespring_invoicespec.csv")
 
-    current_time = timestamp(ts_format="%Y-%m-%d_%H-%M-%S")
+    current_time = dds_web.utils.current_time()
```

Comment:
This should just be `current_time`? or is it standard praxis to refer to things inside the same file with the full path? I just haven't seen it before.